manifesto/README.md

```diff
@@ -1,11 +1,13 @@
-# Kyma Elevator Pitch
+# project Kyma: Elevator Pitch
 
-Kyma is a cloud-native application runtime that gives you the power of Kubernetes but without the hassle of worrying about security, configuration, and integration of tools. 
-Kyma does it for you, as it not only combines various components needed for cloud-native development but also fills the gaps between these components to make them work smoothly together and let you focus on your work.
-With Kyma, you can extend applications using your technology of choice! 
-All you need to do is install and set up Kyma and you’re good to go, with a ready-to-use cluster, and a Console UI, too!
+### _Tagline_<br>
+Extend your applications with Kubernetes
 
-The batteries-included approach eliminates the steep learning curve that comes with Kubernetes.
-It also makes building and running of microservices and serverless functions easier, and service consumption with this approach is just a piece of cake. 
-That's because we handpick the essential tools from the huge CNCF landscape, glue them together adding whatever’s needed to make them play smoothly with each other, ensure security, top it off with a UI, and wrap it up nicely for you as Kyma so you can jump straight to developing applications.
+### _Abstract (200 characters)_<br>
+Kyma *[kee-ma]* is a Kubernetes-based runtime platform for extending existing applications side-by-side with serverless functions and microservices.
```

Comment:
I think Kyma is not **ONLY** for extending applications. We have a group of users that want to use it to build new applications. For those users integrated, secured and upgradeable cloud-native environment is also the value.

Reply:
I fully agree! 
Extension applications, which are the leading scenario for Kyma nowadays, are also applications (although with more specific connectivity requirements), and will certainly benefit from a scalable, lightweight, upgradeable and portable application runtime underneath - absolutely! 

Similarly - Kubernetes itself is not **ONLY** for managing iOT devices, but can as well be used for this ([see Bosch here](https://blog.bosch-si.com/bosch-iot-suite/adopting-kubernetes-to-build-iot-solutions/)). This however did not lead to the Kubernetes project shifting away from its original mission to provide _Production-Grade Container Orchestration_ - it still sticks to it. Instead - Kubernetes project kicked off an [IOT Edge working group](https://www.cncf.io/webinars/introducing-kubernetes-iot-edge-working-group/), to explore that space. 

In the same fashion - Kyma project can certainly establish a working group or a SIG to explore the runtime aspects for applications in there and figure out how to get the project to cater better to those scenarios. This will also give a chance to the users to get involved and provide direct feedback.